.gitignore

```diff
@@ -164,3 +164,6 @@ crashlytics.properties
 crashlytics-build.properties
 fabric.properties
 
+# Vim temp files
```

Comment:
I just found this, might be a more comprehensive .gitignore

https://github.com/github/gitignore/blob/master/Global/Vim.gitignore

Reply:
Most likely, I've never run into a situation where I need more exception, but it would be good to include